apps/ledger-live-desktop/tests/specs/families/ethereum.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from "@playwright/test";
+import test from "../../fixtures/common";
+import { Layout } from "../../models/Layout";
+import { SendModal } from "../../models/SendModal";
+
+test.use({ userdata: "1AccountBTC1AccountETHStarred" });
+
+test("Send flow", async ({ page }) => {
+  const layout = new Layout(page);
+  const sendModal = new SendModal(page);
+
+  await test.step("can open send modal and label should be Max network fees", async () => {
```

Comment:
Maybe more precise with "can open send modal and max network fees label is shown " ?

Reply:
👍  will change it